modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -16359,6 +16358,7 @@ the-connection-to-the-remote-live-site-cannot-be-established-due-to-a-network-pr
 the-content-is-not-valid=The content is not valid.
 the-content-references-a-missing-file-entry=The content references a missing file entry.
 the-content-references-a-missing-page=The content references a missing page.
+the-content-was-imported-but-no-articles-were-found-with-one-of-the-supported-extensions-x=The content was imported, but no articles were found with one of the supported extensions: {0}.
```

Comment:
It's a rule to make separate the commit https://github.com/liferay-lima/liferay-portal/pull/3954/commits/5c5ef6eb8675ee50822eb9625367678ace250063 on 2 separate commits: 

1: The change on the `Language.properties`
2: The build lang on a separate commit for the rest of the languages